inst/examples/app/analytics/app.R

```diff
@@ -17,8 +17,11 @@ library(DT)
 library(shiny.telemetry)
 
 # Default storage backend using LogFile
+log_file_path <- file.path("../instrumentation/user_stats.txt")
```

Comment:
Very minor

We can abstract from the file separator by supplying a vector of the path

![image](https://github.com/Appsilon/shiny.telemetry/assets/211358/f913d155-072a-437d-8238-90d9dba637b1)

```suggestion
log_file_path <- file.path("..", "instrumentation/user_stats.txt")
```